README.md

```diff
@@ -1 +1,4 @@
-# code-review.
\ No newline at end of file
+# code-review.
```

Comment:
hey, great work so far, please use a more descriptive Readme, I would suggest you strictly adhere to the Readme template format provided for you.